test/cmdlineTests/optimizer_inliner_dynamic_reference_constructor/output

```diff
@@ -152,7 +146,6 @@ sub_0: assembly {
     tag_17:
       tag_21
       tag_22
-      jump	// in
```

Comment:
Another one which we'd loose with a more restricted peephole optimizer rule - but well, it's just one byte saved...